e2e/rbd.go

```diff
@@ -527,6 +556,25 @@ var _ = Describe("RBD", func() {
 					if err != nil {
 						e2elog.Failf("failed to create PVC with error %v", err)
 					}
+					app, err := loadApp(appPath)
+					if err != nil {
+						Fail(err.Error())
+					}
+					// write data in PVC
+					app.Namespace = f.UniqueName
+					app.Spec.Volumes[0].PersistentVolumeClaim.ClaimName = pvc.Name
+					wErr := writeDataInPod(app, f)
+					if wErr != nil {
+						Fail(wErr.Error())
+					}
+					opt := metav1.ListOptions{
```

Comment:
you need to driver this Listoptions see https://github.com/ceph/ceph-csi/pull/1464